### PR TITLE
auto cfgs and vars admin field for block cli generation

### PR DESCRIPTION
**Pull request by dev7ch**
This would be great for the extra fields and view files as well, needs reviews and improvement

### CODE REVIEW

File: modules/cms/src/frontend/views/commands/block/create_block.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class <?= $className; ?> extends PhpBlock
      * @var boolean Choose whether block is a layout/container/segmnet/section block or not, Container elements will be optically displayed
      * in a different way for a better user experience. Container block will not display isDirty colorizing.
      */
-    public $isContainer = true;
+    public $isContainer = false;
```
Comment (nadar) [inline modules/cms/src/frontend/views/commands/block/create_block.php]: @dev7ch When the wizzard ask for creating a container, this should be enable. Oterhwise its hidden by default which menas $isContainer = false. 

Short: this must be `true`

File: modules/cms/src/frontend/views/commands/block/create_block.php [hunk 2]
```diff
@@ -123,6 +123,18 @@ public function extraVars()
     */
     public function admin()
     {
-        return '<p><?= $name; ?> Admin View</p>';
+        return  '<p style="display: block; text-align: right; font-weight: bold; text-transform: uppercase;"><?= $name; ?> Admin View</p>'.
```
Comment (nadar) [inline modules/cms/src/frontend/views/commands/block/create_block.php]: Looks good for me. A unit tested would be good.

Comment (dev7ch) [inline modules/cms/src/frontend/views/commands/block/create_block.php]: 'There was 1 failure:

1) luyatests\core\components\FormatterTest::testDefaultSettingsFromGivenArray
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-2017/2017/2017
+2017201720172017/2017201720172017/2017201720172017

/Users/silvan/Sites/luya_dev/luya/tests/core/components/FormatterTest.php:22
/Users/silvan/Sites/luya_dev/luya/vendor/phpunit/phpunit/phpunit:52

FAILURES!
Tests: 204, Assertions: 631, Failures: 1.'

Comment (dev7ch) [inline modules/cms/src/frontend/views/commands/block/create_block.php]: not passing, not clear for me what is corrupt

**Comment by dev7ch** [comment]
It would be wonderful if we could integrate this feature into the v 1.0.0 Realease, is this possible @nadar ? 

I am open and happy to help.

**Comment by dev7ch** [comment]
👍

**Pull request closed without merging**